scripts/test/test-utils.sh

```diff
@@ -134,10 +136,10 @@ run_tests() {
 }
 
 get_pytest_files() {
-  wget -N ${MENDER_ACCEPTANCE_URL}/pytest.ini -P $WORKSPACE/mender-image-tests
```

Comment:
I suggest we just modify `MENDER_ACCEPTANCE_URL` to something like `https://raw.githubusercontent.com/mendersoftware/meta-mender/c791015fb14f27b7b79143494ee50e42f70c4661/tests/acceptance` (last commit before the refactoring.

The reason is that 2.2.x is not prepared to execute the runtime tests (`test_update.py`) and it is not worth the effort.